main.go

```diff
@@ -1,7 +1,14 @@
 package main
 
-import "github.com/cyberark/dev-flow/cmd"
+import (
+	"log"
+	
+	"github.com/cyberark/dev-flow/cmd"
+)
 
 func main() {
+	log.SetPrefix("LOG: ")
```

Comment:
Wouldn't `os.Args[0]` be `df` / `dev-flow`? Seems unnecessary for a CLI. Definitely useful in a system where you have multiple processes dumping into the same log like on the appliance.

Reply:
Yeah program name is marginal utility but I think even removal might be better than pure `LOG:` prefix since on its own it carries no intrinsic meaning and fills out the output. What about `INFO:` or maybe using one of the logging packages that can do logging levels?

Reply:
Just changed this to `INFO:` for now since that does more closely match the intent.